src/app/ApplicationTemplate.Shared/Presentation/Diagnostics/HttpTracingDiagnosticsPageViewModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.ObjectModel;
+using System.Linq;
+using System.Reactive.Linq;
+using ApplicationTemplate.Business;
+using Chinook.DynamicMvvm;
+using Uno;
+
+namespace ApplicationTemplate.Presentation
+{
+	public partial class HttpTracingDiagnosticsPageViewModel : ViewModel
+	{
+		[Inject] private IHttpTracingService _httpTracingService;
+
+		public ObservableCollection<HttpLogItemViewModel> Logs => this.GetObservableCollectionFromObservable(
+			source: _httpTracingService
+				.ObserveLogs()
+				.Select(logs =>
+					logs.Select(log => this.GetChild(() => new HttpLogItemViewModel(log), log.Id.ToString()))
```

Comment:
I fear this creates N ViewModels every time there is an update. After 10 API calls you would have at least 55 ViewModels. If they all complete with a response, you would have 20 updates, so 210 ViewModels, where only 10 are visible.
It should instead only create new ViewModels when items are added to the list (and not recreate the existing ones).

Another problem is that previous ViewModels are not disposed. So for 10 visible ViewModels, you have 200 undisposed ViewModels.

You can use DynamicData to fix this.